### ISSUE TITLE
Sign in 

### ISSUE THREAD

**Issue by heron2014** [brainsandbrawns/native#18]
- [x] create a header with two links - sign in, sign up
- [x] swipe to move to each screens

**Comment by heron2014** [comment]
Swipe from one view to another.
I was trying to use ex-navigation to achieve swiping functionality. The swipe does work when:
- you land on default screen, then you press the button to go to the next screen and on that screen you can swipe to go back. But the problem is that I didn't know how to make swipe available from the first default view, so when you land on default view , you don't need to press the button to move to the second view. I guess the swipe works when you have a stack screens, but it doesn't work from first starting screen. At least I am not sure how to do it right now. (INVESTIGATE FURTHER )

I used react-native-swiper
Current solution to trigger swipe on pressing the button is:
```
<TouchableOpacity onPress={() => this.swiper.scrollBy(1)}>
    <Text>Button</Text>
 </TouchableOpacity>
```

- srollBy api: https://github.com/leecade/react-native-swiper#methods

The only way to go back to previous screen was passing default value to scrollBy which is undefined
(I am not sure if this is the right solution but for now I will leave it as it is )

I got error when I tried to use refs as follow: 
```ref="swiper"```

Reference https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-string-refs.md

**Comment by heron2014** [comment]
Resolved by #19

**Comment by heron2014** [comment]
The above solution for swiper functionality is better to be used as a image swiper but not as part of the routing.  Better option is to use ```react-native-scrollable-tab-view```.
